addons/payment/wizards/payment_capture_wizard_views.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="payment_capture_wizard_view_form" model="ir.ui.view">
+        <field name="name">payment.capture.wizard.form</field>
+        <field name="model">payment.capture.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Capture">
+                <field name="has_draft_children" invisible="1"/>
+                <field name="support_partial_capture" invisible="1"/>
+                <field name="has_remaining_amount" invisible="1"/>
+                <div class="alert alert-warning"
+                     id="alert_draft_capture_tx"
+                     role="alert"
+                     attrs="{'invisible': [('has_draft_children', '=', False)]}">
+                    <p>
+                        <strong>Warning!</strong> There is a partial capture pending. Wait a moment
+                        for it to be processed. If the capture is still pending in a few minutes,
+                        please check your payment provider configuration.
+                    </p>
+                </div>
+                <group>
+                    <field name="transaction_ids" invisible="1"/>
+                    <field name="currency_id" invisible="1"/>
+                    <field name="is_amount_allowed" invisible="1"/>
+                    <field name="authorized_amount"/>
+                    <field name="captured_amount"
+                           attrs="{'invisible': [('captured_amount', '&lt;=', 0)]}"/>
+                    <field name="voided_amount"
+                           attrs="{'invisible': [('voided_amount', '&lt;=', 0)]}"/>
+                </group>
+                <hr/><br/>
+                <group>
+                    <span>
+                        <label for="amount_to_capture" class="oe_inline"/>
+                    </span>
+                    <span>
+                        <field name="amount_to_capture" class="oe_inline"
+                               attrs="{'readonly': [('support_partial_capture', '=', 'full_only')]}"/>
+                        <i class="fa fa-info-circle oe_inline"
+                           title="To change the amount you want to capture, all the transactions must come from providers which supports the partial transaction. If that's not the case but you intended to do a partial capture, manage it directly from the transaction which allows it."/>
+                    </span>
+                    <field name="void_remaining_amount" attrs="{'readonly': [('has_remaining_amount', '=', False)]}"/>
```

Comment:
It was discussed with Dala, and we decided to always show it because the user needs to click somewhere else for this field to appears/disappear whenever it's necessary.